I am a PhD graduate in bioinformatics, a first class honours graduate in computer engineering, an open-source software developer, an avid traveller, a singer, and an enthusiastic cook. I have sixteen years of industry and academic experience developing software in C++, Python, and R, including ten years developing high-performance parallel algorithms for genome sequence assembly, and six years developing embedded and real-time control systems. I developed ABySS 1.0, the first software tool to assemble a human genome from Illumina sequencing, as well as ABySS 2.0, the first assembler to reconstruct a twenty-gigabase conifer genome, seven times the size of the human genome.

## Education and Employment

#### 2019–present—Senior Computational Biologist—10x Genomics

Developed algorithms for the flagship single-cell gene expression software tool, Cell Ranger, which is implemented in Rust and Python.

#### 2012–2019—PhD Bioinformatics—University of British Columbia

NSERC Canada Graduate Scholarship (CGS) student in bioinformatics. Published five papers as the first author. The memory-efficient genome sequence assembly tool ABySS 2.0 (2017) has received over 100 citations. Coursework in bioinformatics, statistics, machine learning, and genetics. GPA 4.26 (93%)

#### 2008–2012—Computational Biologist—BC Cancer Genome Sciences Centre

Developed the genome sequence assembly software package, ABySS, implemented in C++. This heavily parallel and distributed software system, which uses both MPI and OpenMP for parallel computation, was the first system to assemble a human genome using short-read sequencing technology.

#### 2004–2007—Embedded Firmware Developer—Pathway Connectivity

Developed the firmware for Pathway's product line of lighting control devices. These devices, which use Atmel's AVR and ARM microcontrollers, convert between a variety of lighting-control protocols. Modernized the lighting control industry in developing a device, the Pathport, to adapt legacy serial equipment to an Ethernet network. This device won an award for best new product at the industry trade show, Lighting Dimensions International.

#### 1998–2004—BASc Computer Engineering—Simon Fraser University

First class honours scholarship student in computer engineering. Completed an honours undergraduate thesis and two years of co-op work experience at three companies. GPA 3.85

## Expertise

#### Bioinformatics and Genomics

Analysis of DNA and RNA sequencing, including genome sequence assembly, sequence alignment, variant calling, genome annotation, visualization, and inspection of genome assemblies, variants, and rearrangements. Developed ABySS 2.0 to assemble twenty gigabase conifer genomes, seven times the size of human.

#### Data Analysis Pipelines

Extensive experience in constructing data analysis workflows, visualization, reporting, and communicating results effectively with aesthetic appeal.

#### Problem Solving and Leadership

A practiced talent to decompose complex problems into well-defined components, enabling concurrent development by a team. Daily involvement in reviewing code to provide constructive criticism with empathy, to improve both the developer's skills as well as code quality. Use of automated testing to ensure correctness and provide timely automated feedback on code quality to developers.

#### Open-source Developer

Developer and member of the project leadership committee of Homebrew, the package manager for macOS and Linux. Created Homebrew for Linux, and grew its community of fifty thousand users.

#### Sequencing Technologies and Genomic Assays

10x Genomics Chromium, Illumina HiSeq, Oxford Nanopore, Bionano Genomics

#### Programming Languages

Rust, C++, C, Python, R, Ruby, shell, and assembly

#### Programming Methodologies

Functional, generic, and object-oriented programming

#### Parallel Processing

OpenMP, POSIX threads, C++ threads, and MPI

#### Continuous Integration

CircleCI, Travis CI, Azure Pipelines, and GitHub Actions

#### Cloud and Containerization

AWS, Docker, and Singularity

#### Version Control

Git and GitHub wizard. Proven ability to solve any git calamity.

## Selected Publications

A selection from over 30 peer-reviewed publications, h-index of 21, and over 8,000 citations

#### ORCA: A Comprehensive Bioinformatics Container Environment for Education and Research
**SD Jackman**, T Mozgacheva, S Chen, *et al.*
(2019)
*Bioinformatics*

#### Tigmint: Correcting assembly errors using linked reads from large molecules
**SD Jackman**, L Coombe, J Chu, *et al.*
(2018)
*BMC Bioinformatics*

#### ABySS 2.0: Resource-efficient assembly of large genomes using a Bloom filter
**SD Jackman**, BP Vandervalk, H Mohamadi, *et al.*
(2017)
*Genome Research*

#### Organellar Genomes of White Spruce (*Picea glauca*): Assembly and Annotation
**SD Jackman**, RL Warren, EA Gibb, *et al.*
(2015)
*Genome Biology and Evolution*

#### UniqTag: Content-Derived Unique and Stable Identifiers for Gene Annotation
**SD Jackman**, J Bohlmann, I Birol.
(2015)
*PLOS ONE*

#### Assembling the 20 Gb white spruce (*Picea glauca*) genome from WGSS data
I Birol, A Raymond, **SD Jackman**, *et al.*
(2013)
_Bioinformatics_

#### Draft genome of the mountain pine beetle, *Dendroctonus ponderosae* Hopkins
CI Keeling, MM Yuen, ..., **SD Jackman**, *et al.*
(2013)
_Genome Biology_

#### De novo assembly and analysis of RNA-seq data
G Robertson, J Schein, ..., **SD Jackman**, *et al.*
(2010)
_Nature Methods_

#### Assembling genomes using short-read sequencing technology
**SD Jackman**, I Birol
(2010)
_Genome Biology_

#### De novo transcriptome assembly with ABySS
I Birol, **SD Jackman**, CB Nielsen, *et al.*
(2009)
_Bioinformatics_

#### ABySS-Explorer: visualizing genome sequence assemblies
CB Nielsen, **SD Jackman**, I Birol, *et al.*
(2009)
_IEEE Transactions on Visualization and Computer Graphics_

#### ABySS: a parallel assembler for short read sequence data
JT Simpson, K Wong, **SD Jackman**, *et al.*
(2009)
_Genome Research_
